spec/spec_helper.rb

```diff
@@ -18,4 +18,10 @@
     c.syntax = :expect
   end
   config.mock_with :rspec
+
+  config.before(:each) do
+    BundlerApi::GemHelper.any_instance.stub(:set_checksum) do
```

Comment:
This is done already since this syntax is not accepted by rspec 3, which is used in compact_index gem